internal/release/service.go

```diff
@@ -162,27 +162,18 @@ func (s *service) processFilters(ctx context.Context, filters []*domain.Filter,
 
 		l.Info().Msgf("Matched '%s' (%s) for %s", release.TorrentName, release.FilterName, release.Indexer)
 
-		// save release here to only save those with rejections from actions instead of all releases
-		if release.ID == 0 {
-			release.FilterStatus = domain.ReleaseStatusFilterApproved
-
-			if err = s.Store(ctx, release); err != nil {
-				l.Error().Err(err).Msgf("release.Process: error writing release to database: %+v", release)
-				return err
-			}
-		}
-
 		// found matching filter, lets find the filter actions and attach
-		actions, err := s.actionSvc.FindByFilterID(ctx, f.ID)
+		active := true
+		actions, err := s.actionSvc.FindByFilterID(ctx, f.ID, &active)
 		if err != nil {
 			s.log.Error().Err(err).Msgf("release.Process: error finding actions for filter: %s", f.Name)
 			return err
 		}
 
 		// if no actions, continue to next filter
 		if len(actions) == 0 {
-			s.log.Warn().Msgf("release.Process: no actions found for filter '%s', trying next one..", f.Name)
-			return nil
+			s.log.Warn().Msgf("release.Process: no active actions found for filter '%s', trying next one..", f.Name)
+			continue
```

Comment:
@zze0s Do you agree that we should `continue` here? We returned `nil` before, which seems wrong as it does not log the next filter check that way. (It does process it, but no logs from what I can tell)

Reply:
Correct, continue will make it check the next filter then which is the intended way.